### PR TITLE
🌐 Add Korean translation for `docs/ko/docs/tutorial/handling-errors.md`

### PR DESCRIPTION
**Pull request by NinaHwang**
This PR translates tutorial/handling-errors.md in Korean.

related: #2017

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit f99c1137f091e4ccaf4ca70631dfabb38a85c280 at: https://6499e3571387310549a3625c--fastapi.netlify.app

**Comment by tiangolo** [comment]
Thanks you! I'm still waiting for others to review this to be able to merge it, meanwhile, maybe you could help me review some of the other Korean translations, that way if they are approved by each other, I could merge them. :nerd_face: 

https://github.com/tiangolo/fastapi/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc+label%3Alang-ko+label%3Aawaiting-review

File: docs/ko/docs/tutorial/handling-errors.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+{!../../../docs_src/handling_errors/tutorial001.py!}
+```
+
+### 코드에서 `HTTPException` 발생시키기
+
+`HTTPException`은 API에 대한 추가적인 데이터를 포함한 일반적인 파이썬 예외입니다.
+
+파이썬 예외이기 때문에, 반환(`return`)하지 않고 발생(`raise`)시킵니다.
+
+이것은 만약 당신이 *경로 동작 함수*의 내부에서 호출하는 유틸리티 함수 내부에 있고, 해당 유틸리티 함수 내부에서 `HTTPException`을 발생시키는 경우, *경로 동작 함수*의 나머지 부분을 실행하는 대신 즉시 요청에 대한 작업을 중단하고 `HTTPException`에 따른 HTTP 오류를 클라이언트에게 전송한다는 것을 의미합니다.
```
Comment (junah201) [inline docs/ko/docs/tutorial/handling-errors.md:38]:
```suggestion
이것은 만약 당신이 *경로 작동 함수*의 내부에서 호출하는 유틸리티 함수 내부에 있고, 해당 유틸리티 함수 내부에서 `HTTPException`을 발생시키는 경우, *경로 작동 함수*의 나머지 부분을 실행하는 대신 즉시 요청에 대한 작업을 중단하고 `HTTPException`에 따른 HTTP 오류를 클라이언트에게 전송한다는 것을 의미합니다.
```

In https://github.com/tiangolo/fastapi/discussions/3167 , We have agreed to translate `Path operation` to`경로 작동`. So, `경로 작동` is better than `경로 동작`.

File: docs/ko/docs/tutorial/handling-errors.md [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+
+그리고 당신은 FastAPI를 사용하여 해당 예외를 전역적으로 처리하고자 합니다.
+
+`@app.exception_handler()`를 사용하여 사용자 예외를 추가할 수 있습니다:
+
+```Python hl_lines="5-7  13-18  24"
+{!../../../docs_src/handling_errors/tutorial003.py!}
+```
+
+여기서 `/unicorns/yolo`를 요청하면, *경로 동작*은 `UnicornException`을 발생(`raise`)시킬 것입니다.
```
Comment (junah201) [inline docs/ko/docs/tutorial/handling-errors.md:99]:
```suggestion
여기서 `/unicorns/yolo`를 요청하면, *경로 작동*은 `UnicornException`을 발생(`raise`)시킬 것입니다.
```

**Comment by tiangolo** [comment]
As this PR had requested changes to be applied but has been inactive for a while, it's now going to be closed. But if there's anyone interested, feel free to create a new PR.

**Pull request closed without merging**